src/utils/utils.ts

```diff
@@ -31,3 +33,26 @@ export async function executeCommand(command: string, cwd?: string): Promise<str
 export function generateQuotedFullName(schema: string, objectName: string): string {
 	return `[${escapeClosingBrackets(schema)}].[${escapeClosingBrackets(objectName)}]`;
 }
+
+
+/**
+ * Copied from Azure function: Create Function
+ * https://github.com/microsoft/vscode-azurefunctions/blob/main/src/commands/createFunction/FunctionNameStepBase.ts
+ * @param folderPath selected azure project folder path
+ * @param functionName objectName that was chosen by the user
+ * @returns the function name that will always add at least `1` to the function name
+ */
+export async function getUniqueFsPath(folderPath: string, functionName: string): Promise<string | undefined> {
```

Comment:
This function name isn't accurate - it's not getting the path but rather just the unique filename (without the extension)